readme.md

```diff
@@ -3,30 +3,47 @@
 # AMP Service worker
 AMP service worker is a service worker library for your AMP pages.
 
-The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
-Core use cases:
+**Purpose:** The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
+
+**Core use cases:**  
 - Cache AMP scripts with a `stale-while-revalidate` strategy for a longer duration that http-headers.
 - Cache valid visited AMP documents, and serve only in case of flaky network conditions.
 - Cache assets which are critical to a page with a given strategy.
-- Prefetch outgoing links from an AMP page.
 - Cache an offline page in order to show when a user navigates to a page which was previously not visited.
+- Prefetch outgoing links from an AMP page.
 
 
 ## Usage
-In order to use this library user can include the library with an importScript in their service worker
+In order to use AMP-SW you need to add a service worker to your AMP pages. You can follow these steps to do the same and add network resiliency to your AMP pages.
 
+- Host a service worker file at the same directory level as your AMP web app/ AMP page.
+- Add [amp-install-serviceworker](https://amp.dev/documentation/components/amp-install-serviceworker) to your AMP pages and point it your service worker file.
```

Comment:
For each of these steps and example would be beneficial.

Reply:
done